### ISSUE TITLE
Comment Social Reactions Error

### ISSUE THREAD

**Issue by shadowdare** [shadowdare/Vanilla-App-Articles#34]
When clicking on a Twitter social reaction link under comments, a page saying "Whoops! I don't know what a is" is displayed.

See: http://vanillaforums.org/discussion/29234/yaga-reactions-vanilla-social-reactions-integration

**Comment by shadowdare** [comment]
After looking into this issue, here's what I found:

- The Twitter/Facebook sharing icon that shows up below an article comment comes from the Twitter plugin and Facebook plugin that comes with Vanilla. These plugins can be enabled from the "Social" dashboard settings page below the "Forum" section, as these plugins require the Vanilla app to be enabled.

- I only looked into the Twitter plugin, but I guess the Facebook plugin works similarly. The Twitter plugin has a method in `class.twitter.plugin.php` declared with `PostController_Twitter_Create($Sender, $RecordType, $ID)` that makes a call to a function with `GetRecord($RecordType, $ID)`.

- The GetRecord() function is defined in `functions.general.php` and only supports the `discussion` and `comment` types, which are from the Vanilla app. There isn't a way to add support for the `article` and `articlecomment` types without overriding the function.

Finally, I won't be providing support for the official Twitter and Facebook plugins that comes with Vanilla since those are meant for the Vanilla app's discussions and comments.

I'll rename the event where the social reaction icons show up under the comment to something else.
